libs/application/templates/parental-leave/src/fields/OtherParent/index.tsx

```diff
@@ -8,38 +8,126 @@ import {
 import { RadioFormField } from '@island.is/application/ui-fields'
-import { getOtherParentOptions } from '../../lib/parentalLeaveUtils'
-import { SPOUSE, NO, SINGLE } from '../../constants'
+import {
+  getApplicationAnswers,
+  getOtherParentOptions,
+} from '../../lib/parentalLeaveUtils'
+import { SPOUSE, NO, SINGLE, TransferRightsOption } from '../../constants'
 import { useFormContext } from 'react-hook-form'
 
 export const OtherParent: FC<FieldBaseProps> = ({ application, field }) => {
   const { id, title } = field
+  const { errors, setValue, register } = useFormContext()
   const { formatMessage } = useLocale()
-  const { errors, setValue } = useFormContext()
+
+  // const {
+  //   transferRights,
+  //   isRequestingRights,
+  //   requestDays,
+  //   isGivingRights,
+  //   giveDays,
+  //   otherParentRightOfAccess,
+  // } = getApplicationAnswers(application.answers)
+  // const [choseTransferRights, setTransferRights] = useState(transferRights)
+  // const [choseRequestingRights, setRequestingRights] = useState(
+  //   isRequestingRights,
+  // )
+  // const [choseRequestDays, setRequestDays] = useState(requestDays)
+  // const [choseGivingRights, setGivingRights] = useState(isGivingRights)
+  // const [choseGiveDays, setGiveDays] = useState(giveDays)
+  // const [choseOtherParentRightOfAccess, setOtherParentRightOfAccess] = useState(
+  //   otherParentRightOfAccess,
+  // )
+
   return (
-    <Box>
-      <Text variant="h4" as="h4">
-        {formatText(title, application, formatMessage)}
-      </Text>
-      <RadioFormField
-        error={errors && getErrorViaPath(errors, id)}
-        field={{
-          id: id,
-          type: FieldTypes.RADIO,
-          component: FieldComponents.RADIO,
-          title,
-          children: undefined,
-          options: (application) =>
-            getOtherParentOptions(application, formatMessage),
-          onSelect: (s: string) => {
-            if (s === SPOUSE || s === NO || s === SINGLE) {
-              setValue('otherParentObj.otherParentName', '')
-              setValue('otherParentObj.otherParentId', '')
-            }
-          },
-        }}
-        application={application}
-      />
-    </Box>
+    <>
+      <Box>
+        <Text variant="h4" as="h4">
+          {formatText(title, application, formatMessage)}
+        </Text>
+        <RadioFormField
+          error={errors && getErrorViaPath(errors, id)}
+          field={{
+            id: id,
+            type: FieldTypes.RADIO,
+            component: FieldComponents.RADIO,
+            title,
+            children: undefined,
+            options: (application) =>
+              getOtherParentOptions(application, formatMessage),
+            onSelect: async (s: string) => {
+              if (s === SPOUSE || s === NO || s === SINGLE) {
+                setValue('otherParentObj.otherParentName', '')
+                setValue('otherParentObj.otherParentId', '')
+                // if (s === NO || s === SINGLE) {
```

Comment:
This as well, and on other places.

Reply:
Same here ( I have them there so if someone want to take this feature could continue to work with it )